app/src/main/java/com/hyperion/ths/marvel_03/ui/OnItemClickListener.java

```diff
@@ -0,0 +1,15 @@
+package com.hyperion.ths.marvel_03.ui;
+
+/**
+ * Created by ths on 07/06/2017.
+ */
+
+import com.hyperion.ths.marvel_03.data.model.Hero;
+import com.hyperion.ths.marvel_03.ui.BaseRecyclerView;
```

Comment:
ERROR: Redundant import from the same package - com.hyperion.ths.marvel_03.ui.BaseRecyclerView.